apps/judicial-system/backend/src/app/modules/verdict/verdict.controller.ts

```diff
@@ -16,69 +16,76 @@ import {
   RolesGuard,
   RolesRules,
 } from '@island.is/judicial-system/auth'
-import { ServiceRequirement } from '@island.is/judicial-system/types'
+import { indictmentCases } from '@island.is/judicial-system/types'
 
 import {
   districtCourtAssistantRule,
   districtCourtJudgeRule,
   districtCourtRegistrarRule,
+  publicProsecutorStaffRule,
 } from '../../guards'
 import {
   Case,
+  CaseCompletedGuard,
+  CaseExistsGuard,
+  CaseReadGuard,
+  CaseTypeGuard,
   CurrentCase,
-  MinimalCaseAccessGuard,
-  MinimalCaseExistsGuard,
 } from '../case'
+import { DefendantExistsGuard } from '../defendant'
 import { UpdateVerdictDto } from './dto/updateVerdict.dto'
-import { ValidateVerdictGuard } from './guards/validateVerdict.guard'
 import { CurrentVerdict } from './guards/verdict.decorator'
-import { VerdictWriteGuard } from './guards/verdictWrite.guard'
+import { VerdictExistGuard } from './guards/verdictExistGuard.guard'
 import { Verdict } from './models/verdict.model'
 import { VerdictService } from './verdict.service'
 
-@Controller('api/case/:caseId/defendant/:defendantId/verdict')
+@Controller('api/case/:caseId/defendant/verdict')
 @ApiTags('verdicts')
 @UseGuards(
   JwtAuthUserGuard,
   RolesGuard,
-  MinimalCaseExistsGuard,
-  MinimalCaseAccessGuard,
-  ValidateVerdictGuard,
+  CaseExistsGuard,
+  new CaseTypeGuard(indictmentCases),
+  CaseReadGuard,
+  CaseCompletedGuard,
+  DefendantExistsGuard,
+  VerdictExistGuard,
 )
 export class VerdictController {
   constructor(
     private readonly verdictService: VerdictService,
     @Inject(LOGGER_PROVIDER) private readonly logger: Logger,
   ) {}
 
-  @UseGuards(VerdictWriteGuard)
   @RolesRules(
     districtCourtJudgeRule,
     districtCourtRegistrarRule,
     districtCourtAssistantRule,
+    publicProsecutorStaffRule,
   )
-  @Patch(':verdictId')
+  @Patch(':defendantId')
```

Comment:
This works, but I would probably do .../defendant/:defendantId/verdict. That is more consistent with paths used in other controllers, for example the case controller. Still, I don't have a strong opinion.